onboarder/installation/define_environment_variables/dot_env_component.rst

```diff
@@ -3,7 +3,21 @@ Using the DotEnv file
 
 The `Dotenv Symfony component <https://github.com/symfony/dotenv>`_ is a dependency already available
 into the PIM as it's a standard Symfony framework component. It is also used by default in the PIM starting Akeneo PIM
-``4.0``.
+``>= 4.0``.
 
 Edit the ``.env`` file at your project root directory and add the environment variables values provided on your
 Onboarder project page on the Partner Portal.
+
+.. code-block:: bash
```

Comment:
I don't understand why you are adding **Onboarder** env variables to the pim-docs. Customers don't have access to those env variables.